### PR TITLE
Created Login UI

### LINKED ISSUE #31: [Feature 5] User Login System
Letting a user logn

### PR DESCRIPTION
**Pull request by Thishor**
Implemented Login UI and Register UI

### CODE REVIEW

**Comment by CSEndeavour** [comment]
Finish implementing the database to store users info quickly.

**Comment by CSEndeavour** [comment]
Implement a way to store the user that's currently logged in. Consider using SharedPreferences to do this.

**Comment by CSEndeavour** [comment]
Merge your code into MainActivity.

File: app/src/main/java/com/example/notes2text/myapplication/RegisterView.java [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+                Intent intent = new Intent(RegisterView.this, LoginView.class);
+                startActivity(intent);
+            }
+        });
+    }
+
+
+    public void clickRegister(View view) {
+    }
+}
```
Comment (CSEndeavour) [inline app/src/main/java/com/example/notes2text/myapplication/RegisterView.java:76]: Remember to add this code the next time you make a pull request.

**Review by CSEndeavour — APPROVED** [review]

**Review by julianmarrero — APPROVED** [review]
Add comments to files to explain the purpose of methods and/or explain specific steps within methods that are unclear.

File: app/src/main/java/com/example/notes2text/myapplication/LoginView.java [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+                startActivity(intent);
+            }
+        });
+    }
+
+    public void loginClick(View view){
+        LoginUseCase loginAttempt = new LoginUseCase();
+        EditText username = findViewById(R.id.username);
+        EditText password = findViewById(R.id.password);
+        if (loginAttempt.checkUser(username.getText().toString(), password.getText().toString()))
```
Comment (JerryLi-617) [inline app/src/main/java/com/example/notes2text/myapplication/LoginView.java]: Seem to have a syntax error with the if statement, missing "{"

File: app/src/main/java/com/example/notes2text/myapplication/User.java [hunk 3]
```diff
@@ -0,0 +1,13 @@
+package com.example.notes2text.myapplication;
+
+public class User {
+    protected String username;
+    protected String password;
```
Comment (JerryLi-617) [inline app/src/main/java/com/example/notes2text/myapplication/User.java:5]: For more security on password, consider making password private, and have a protected function to check user input against stored password.

**Review by JerryLi-617 — APPROVED** [review]

**Pull request merged**